lib/components/app/locale-selector.tsx

```diff
@@ -21,34 +21,35 @@ const LocaleSelector = (props: LocaleSelectorProps): JSX.Element | null => {
 
   // Only render if two or more languages are configured.
   return languageOptions ? (
-    <Dropdown
-      id="locale-selector"
-      label={intl.formatMessage({ id: 'components.SubNav.selectALanguage' })}
-      listLabel={intl.formatMessage({ id: 'components.SubNav.languages' })}
-      name={
-        <span
-          style={{
-            color: 'rgba(255, 255, 255, 0.85)'
-          }}
-        >
-          <GlobeAmericas height="18px" />
-        </span>
-      }
-      style={{ display: 'block ruby' }}
-      // TODO: How to make this work without block ruby?
-    >
-      {Object.keys(languageOptions).map((locale: string) => (
-        <li key={locale} lang={locale} role="none">
-          <UnstyledButton
-            aria-selected={locale === currentLocale || undefined}
-            onClick={() => setLocale(locale)}
-            role="option"
+    <li>
```

Comment:
I think this might be the reason I can no longer open the dropdown using keyboard navigation

Reply:
Good catch! Fixed this with a `preventDefault` inside `dropdown`!